package.json

```diff
@@ -64,7 +64,7 @@
     "dateformat": "3.0.3",
     "express": "~4.16.4",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "logops": "2.1.0",
+    "logops": "2.1.2",
```

Comment:
We are fixing this in master through a separate PR in each IOTA (e.g. https://github.com/telefonicaid/iotagent-json/pull/608). I'd suggest to wait for its merge, then upgrade your PRs synching from master.

Reply:
Merges completed.